charts/cmc-claim-store/values.aat.template.yaml

```diff
@@ -0,0 +1,10 @@
+claim-store:
+  environment:
+    FRONTEND_BASE_URL: https://moneyclaims.aat.platform.hmcts.net
+    IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    SEND_LETTER_URL: http://rpe-send-letter-service-aat.service.core-compute-aat.internal
+    RESPOND_TO_CLAIM_URL: https://moneyclaims.aat.platform.hmcts.net/first-contact/start
+    IDAM_API_URL: https://preprod-idamapi.reform.hmcts.net:3511
+    PDF_SERVICE_URL: http://cmc-pdf-service-aat.service.core-compute-aat.internal
+#    CCD_GATEWAY_HOST: http://${SERVICE_NAME}-api-gateway
+#    SERVICES_CORECASEDATA_BASEURL : http://${SERVICE_NAME}-data-store-api
```

Comment:
what's this?

Reply:
pls ignore these for now - i will fix with next PR (coming soon)